docs/constants/navigation.js

```diff
@@ -237,36 +237,38 @@ export const general = [
   makeSection('Expo Router', [
     makePage('router/introduction.mdx'),
     makePage('router/installation.mdx'),
-    makePage('router/create-pages.mdx'),
-    makePage('router/navigating-pages.mdx'),
-    makePage('router/layouts.mdx'),
-    makePage('router/error-handling.mdx'),
+    makeGroup('Router 101', [
+      makePage('router/basics/core-concepts.mdx'),
+      makePage('router/basics/notation.mdx'),
+      makePage('router/basics/layout.mdx'),
+      makePage('router/basics/navigation.mdx'),
+      makePage('router/basics/real-life-nav.mdx'),
+    ]),
     makeGroup('Navigation patterns', [
-      makePage('router/advanced/root-layout.mdx'),
       makePage('router/advanced/stack.mdx'),
       makePage('router/advanced/tabs.mdx'),
       makePage('router/advanced/drawer.mdx'),
+      makePage('router/reference/authentication.mdx'),
```

Comment:
```suggestion
      makePage('router/reference/authentication.mdx'),
```

Let's migrate this file to "advanced" section and add a redirect. Can you also add redirects in `error-utilties.ts` and `deploy.sh` files for other removed pages?

Reply:
I think I got them all and they look right to me, but not sure how to test locally (didn't seem to work there). Can you check them?